### PR TITLE
NR-83104 : run infra with terraform

### PR DESCRIPTION
**Pull request by brushknight**

### CODE REVIEW

File: .github/workflows/ci-prerelease.yml [hunk 1]
```diff
@@ -89,3 +89,11 @@ jobs:
           package_name: ${{ env.NR_DISTRO }}
           package_version: ${{ env.NR_VERSION }}
           platforms: "al2,al2022,centos7,centos8,debian-bullseye,debian-buster,redhat8,redhat9,suse15.2,suse15.3,suse15.4,ubuntu1604,ubuntu1804,ubuntu2004,ubuntu2204"
+
+    packaging-tests:
```
Comment (rubenruizdegauna) [inline .github/workflows/ci-prerelease.yml:93]: this doesn't seem properly indented. Is it working? 🤔

File: test/provision/terraform/main.tf [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+provider "aws" {
+  region = var.region
+}
+
+variable "inventory_output" {
+  default = "./inventory.ec2"
+}
+
+module "otel-env-provisioner" {
+  source             = "git::https://github.com/newrelic-experimental/otel-env-provisioner//terraform/otel-ec2?ref=otel_ec2"
```
Comment (brushknight) [inline test/provision/terraform/main.tf:33]: use main here

File: test/packaging/ansible/upgrade.yaml [hunk 3]
```diff
@@ -61,9 +61,9 @@
           assert:
             that: collector_pid.stdout != updated_collector_pid.stdout
 
-      always:
-        - name: Final cleanup # Only required for shared infra.
-          include_role:
-            name: cleanup
+      # always:
```
Comment (rubenruizdegauna) [inline test/packaging/ansible/upgrade.yaml:64]: this should be uncommented, shouldn't it?

**Review by rubenruizdegauna — APPROVED** [review]
LGTM!  just one leftover ;-)

**Review by rubenruizdegauna — APPROVED** [review]
LGTM! 🔥

**Review by rubenruizdegauna — APPROVED** [review]
LGTM

**Pull request merged**